docs/reference/ilm/policy-definitions.asciidoc

```diff
@@ -0,0 +1,505 @@
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+
+===== Example: Assign index to a specific node and update replica settings
+
+This example updates the index to have one replica per shard and be allocated
+to nodes with a `box_type` attribute of "cold".
+
+[source,js]
+--------------------------------------------------
+PUT _ilm/policy/my_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "actions": {
+          "allocate" : {
+            "number_of_replicas": 1,
+            "require" : {
+              "box_type": "cold"
+            }
+        }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+
+[[ilm-delete-action]]
+==== Delete
+
+Phases allowed: delete.
+
+The Delete Action does just that, it deletes the index.
+
+This action does not have any options associated with it.
+
+[source,js]
+--------------------------------------------------
+PUT _ilm/policy/my_policy
+{
+  "policy": {
+    "phases": {
+      "delete": {
+        "actions": {
+          "delete" : { }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+
+[[ilm-forcemerge-action]]
+==== Force Merge
+
+Phases allowed: warm.
+
+NOTE: Index will be be made read-only when this action is run (see: <<dynamic-index-settings,index.blocks.write>>)
+
+The Force Merge Action <<indices-forcemerge,force merges>> the index into at most a specific number of <<indices-segments,segments>>.
+
+[[ilm-forcemerge-options]]
+.Force Merge Options
+[options="header"]
+|======
+| Name                 | Required  | Default             | Description
+| `max_num_segments`   | yes       | -                   | The number of segments to merge to. To fully merge the index, set it to `1`
+|======
+
+[source,js]
+--------------------------------------------------
+PUT _ilm/policy/my_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "actions": {
+          "forcemerge" : {
+            "max_num_segments": 1
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+
+[[ilm-readonly-action]]
+==== Read-Only
+
+Phases allowed: warm.
+
+This action will set the index to be read-only (see: <<dynamic-index-settings,index.blocks.write>>)
+
+This action does not have any options associated with it.
+
+[source,js]
+--------------------------------------------------
+PUT _ilm/policy/my_policy
+{
+  "policy": {
+    "phases": {
+      "warm": {
+        "actions": {
+          "readonly" : { }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+
+[[ilm-rollover-action]]
+==== Rollover
+
+Phases allowed: hot.
+
+[WARNING]
+index format must match pattern '^.*-\\d+$', for example (`logs-000001`).
+[WARNING]
+The managed index must set `index.lifecycle.rollover_alias` as the
+alias to rollover. The index must also have this alias pointing to it.
```

Comment:
`The index must also have this alias pointing to it` - I think we need to explain more here, because its not enough that the alias points to the index, the index either needs to be the only index int he alias or it needs to be the write index for the alias.

Reply:
good call. I've added a link to the section in the rollover docs talking about this behavior, as well as an example for creating a managed index with an alias and a rollover_alias setting set.